admin/common_auth/models.py

```diff
@@ -73,7 +73,14 @@ class Meta:
     #     return self.is_admin
 
     @property
+    def full_name(self):
```

Comment:
They're defined in `common_auth.models.MyUser`. I'd talk it over with @Ghalko or @zkraime but I'd wager that they're used for displaying things to the user and it's probably fine to change them. Especially since if they need the email for something programmatic or for display in a template u.email will still work.